orders-war/src/test/java/au/gov/qld/pub/orders/dao/NoticeToPayDAOIT.java

```diff
@@ -5,29 +5,29 @@
 import static org.hamcrest.Matchers.is;
 
 import org.joda.time.DateTime;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
 
 import au.gov.qld.pub.orders.ApplicationContextAwareTest;
 import au.gov.qld.pub.orders.entity.NoticeToPay;
 import au.gov.qld.pub.orders.service.PaymentInformationService;
 
-public class NoticeToPayDAOTest extends ApplicationContextAwareTest {
+public class NoticeToPayDAOIT extends ApplicationContextAwareTest {
```

Comment:
Would it still be ideal to keep "Test" in the Class name?

Reply:
https://maven.apache.org/surefire/maven-failsafe-plugin/examples/junit-platform.html#filtering-by-test-class-names-for-maven-failsafe

The Maven Failsafe Plugin will scan for test classes whose fully qualified names match the following patterns.
```
**/IT*.java
**/*IT.java
**/*ITCase.java
```
Moreover, it will exclude all nested classes (including static member classes) by default.

https://stackoverflow.com/questions/30077033/is-the-it-java-filename-suffix-instead-of-test-java-for-junit-integration
Note, however, that you can override this default behavior by configuring explicit `include` and `exclude` rules in your `pom.xml` file. For example, to keep Maven Failsafe from excluding static member classes, you can override its exclude rules.


The IT.java suffix is a convention used by the [Maven Failsafe Plugin](https://maven.apache.org/surefire/maven-failsafe-plugin/). All classes in the test directory that have the suffix IT are executed by this plugin in the integragion-test phase. (Tests with suffix Test are exectued by the [Maven Surefire Plugin](https://maven.apache.org/surefire/maven-surefire-plugin/index.html) in the test phase.)